shared/naturalcrit/splitPane/splitPane.jsx

```diff
@@ -139,7 +139,7 @@ const SplitPane = createClass({
 
 	render : function(){
 		return <div className='splitPane' onPointerMove={this.handleMove} onPointerUp={this.handleUp}>
-			<Pane
+			<Pane className="editorPane"
```

Comment:
Should these be named `leftPane` and `rightPane`, since they are now being used for `vault`, etc.

Reply:
probably